src/SDKs/RedisCache/Management.Redis/Microsoft.Azure.Management.Redis.csproj

```diff
@@ -6,7 +6,7 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.Redis</PackageId>
 		<Description>Provides redis cache management capabilities for Microsoft Azure.</Description>
-		<Version>4.3.0-preview</Version>
+		<Version>4.4.0</Version>
 		<AssemblyName>Microsoft.Azure.Management.Redis</AssemblyName>
 		<PackageTags>Microsoft Azure Redis Cache management;Redis Management;</PackageTags>
 		<PackageReleaseNotes/>
```

Comment:
Missing package release notes for this release. If this release has lot of new features, this will be a good time to document all those new features in the release, so that users see what is included in the nuget package.